tutorials/basic_1_square/README.md

```diff
@@ -0,0 +1,137 @@
+# Basic: Rendering a simple square on the screen
+
+This tutorial will introduce you to:
+
+ - `Game`: The basic class to build a game on, you will understand its structure and how to use it to build a simple game
+ - `GameWidget`: The widget used to place your game on your widget tree
+ - Basic rendering: You will get introduced to the basics of using `dart:ui` class `Canvas` which is the class which Flame uses for rendering your game.
+
+All the code of this tutorial code can be found [here](./code) and is based on Flame `1.0.0-rc7` version.
+
+By the end of this tutorial you will have built a simple game that renders a square bouncing on the screen that will look like this:
+
+![Preview](./media/preview.gif)
+
+## Building your first game
+
+`Game` is the most basic class that you can use to build your game, it includes the necessary methods for creating a basic game loop and methods for the lifecycle of a Flame game.
+
+`GameWidget` is, like its name suggest, a Flutter widget that will run your game and place it inside the Flutter widget tree.
+
+As a first step, lets just put a Game instance running, for that we will need to create our own class that extends Flame's Game class, implement its methods, and pass an instance of that Game to a GameWidget, somehting like this:
+
+```dart
+import 'package:flutter/material.dart';
+import 'package:flame/game.dart';
+
+void main() {
+  runApp(GameWidget(
+    game: MyGame(),
```

Comment:
Is it not standard to put the creation of a `Game` into it's own variable, to ensure it wont be accidentally rebuilt? So that people will be more prone to do that later on as well?